pkg/kubelet/images/image_manager.go

```diff
@@ -219,3 +322,57 @@ func applyDefaultImageTag(image string) (string, error) {
 	}
 	return image, nil
 }
+
+// isEnsuredBySecret - returns true if the secret for an auth used to pull an
```

Comment:
can we update the comment here since it returns two boolean values?

Reply:
Updated.